src/Foundation/NSOperatingSystemVersion.cs

```diff
@@ -36,20 +36,102 @@ namespace Foundation  {
 	[SupportedOSPlatform ("macos")]
 	[SupportedOSPlatform ("tvos")]
 #endif
-	// NSProcessInfo.h
-	public struct NSOperatingSystemVersion {
+
+	public struct NSOperatingSystemVersion: IEquatable<NSOperatingSystemVersion>, IComparable{
```

Comment:
```suggestion
	public struct NSOperatingSystemVersion : IEquatable<NSOperatingSystemVersion>, IComparable{
```
A space before the colon would also be preferable here :)